gildniy/WayFarer
# #167846555 Ft create trips table db

#### What does this PR do?
Create trips table
#### Description of Task to be completed?
Trips table created in the PostgreSQL database
#### How should this be manually tested?
After cloning the project, and install the dependencies, you can run `npm run create` and should see that the `trips` table was created in pgAdmin
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#167846555
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
